client/reader/daily-post/index.jsx

```diff
@@ -91,10 +94,14 @@ class DailyPostButton extends React.Component {
 	}
 
 	pickSiteToPostTo( siteSlug ) {
+		const pingbackAttributes = getPingbackAttributes( this.props.post );
+
 		recordAction( 'daily_post_challenge' );
 		recordGaEvent( 'Clicked on Daily Post challenge' );
 		recordTrackForPost( 'calypso_reader_daily_post_challenge_site_picked', this.props.post );
-		const pingbackAttributes = getPingbackAttributes( this.props.post );
+
+		markPostSeen( this.props.post );
```

Comment:
Adding this to record a post view if the user clicks the "Post About Abc" from a feed.